infra/tests/tests_views.py

```diff
@@ -0,0 +1,140 @@
+'''Unit tests for auth views.'''
+from django.contrib.auth import get_user_model
+from django.utils.timezone import now
+from django.urls import reverse
+from model_mommy import mommy
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+import infra.models
+
+
+User = get_user_model()
+
+
+class TestOperationLogViewSet(APITestCase):
```

Comment:
As I commented in other comments, OperationLog is not handled by front-end users and we do not expose any routes for it, so delete this test.